3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -18,18 +18,77 @@ Complete the four functions provided in the starter `index.js` file:
 Try and avoid using global variables. Instead, use function parameters and 
 return values to pass data back and forth.
 ------------------------------------------------------------------------------*/
-function fetchData(/* TODO parameter(s) go here */) {
-  // TODO complete this function
+async function fetchData(url) {
+  const data = await fetch(url);
+  const response = await data.json();
```

Comment:
You're not handing api errors here (http status codes other than 2xx), could be nice to use the same snipper of code you wrote in the previous exercise here too ;)